ci/ray_ci/bisect/validator.py

```diff
@@ -0,0 +1,16 @@
+import abc
+
+
+class Validator(abc.ABC):
+    def __init__(
+        self,
+        test: str,
+    ) -> None:
+        self.test = test
```

Comment:
you should not need this `__init__`. constructor is typically not part of the abstract class.

do you intend to have one validator for each test? I thought it should be one validator for each type of tests.

Reply:
yes, moved 'test' into the 'run' function